examples/topic_models/lntm_mcem.py

```diff
@@ -167,6 +222,22 @@ def e_obj(observed):
                           np.mean(accs), np.mean(Eta_mean),
                           np.mean(Eta_logstd)))
 
+        # Run AIS
+        print('Start evaluation! {} temperatures, {} chains'
+              .format(_n_temperatures, _n_chains))
+        time_ais = -time.time()
+
+        ll_lb = _ais.run(sess, feed_dict={_x: X_test,
+                                          eta_mean: Eta_mean,
+                                          eta_logstd: Eta_logstd})
+
+        time_ais += time.time()
+
```

Comment:
Maybe we need an empty run to initialize the step size?

Or we can make _hmc's step size as a placeholder, and turn off the adaptation. We feed step_size=hmc_info.step_size here.